sample/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 27
+
+
+
+    defaultConfig {
```

Comment:
nit: lots of empty lines. Also let's add an empty line before the `android` block.